### PR TITLE
Links

### PR DESCRIPTION
**Pull request by Chloejay**
Add Apache Cassandra and HBase to Datastores and link to blog posts.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -25,6 +25,9 @@ toc::[]
 * https://pinot.apache.org/[Apache Pinot] - A realtime distributed OLAP datastore.
 * https://www.influxdata.com/[InfluxDB] - Purpose-Built Open Source Time Series Database.
 * https://www.postgresql.org/[Postgres] - The World's Most Advanced Open Source Relational Database.
+* http://cassandra.apache.org/[Apache Cassandra] - Open Source distributed wide column store, NoSQL database.
```
Comment (gunnarmorling) [inline README.adoc]: Nice! Could you only add them in alphabetical order? Thanks!

File: README.adoc [hunk 2]
```diff
@@ -88,8 +91,7 @@ _only overview contents, no specific tools_
 * https://www.dataengineeringpodcast.com/[Data Engineering Podcast]
 
 === Blog Posts and Articles
-
-tbd.
+* https://dataengweekly.substack.com/[Data Eng Weekly]
```
Comment (gunnarmorling) [inline README.adoc]: Cool, didn't know that one yet!

File: README.adoc (hunk 1 above)
Comment (Chloejay) [reply README.adoc]: yes, just reordered on datastores section.

File: README.adoc (hunk 2 above)
Comment (Chloejay) [reply README.adoc]: Thanks, this blog quality is good.

**Comment by gunnarmorling** [comment]
Thanks a lot!

**Comment by Chloejay** [comment]
Thank you too for creating this nice awesome project! look forward to seeing it grows and using it as my learning reference!

**Pull request merged**